packages/commonwealth/client/scripts/views/components/NewThreadFormLegacy/NewThreadForm.tsx

```diff
@@ -413,6 +425,36 @@ export const NewThreadForm = forwardRef<
         });
 
         const thread = await createThread(input);
+
+        if (tokenizedThreadsAllowed?.tokenized_threads_enabled) {
+          if (!communityToken?.token_address) {
+            notifyError('Community token not found');
+            return;
+          }
+
+          if (!community?.ChainNode?.id) {
+            notifyError('chainId not found');
+            return;
+          }
+
+          await createThreadToken({
```

Comment:
should this only be possible from new thread form? bc we can also create thread from sticky input, check `CommunityHomePage => handleCreateThread` and `DiscussionsPage => handleSubmitThread`.